.circleci/config.yml

```diff
@@ -175,66 +172,89 @@ jobs:
             mkdir docs
             cd docs
             python ../.circleci/docs-update-grakn-dependency-on-dev-branch.py
+  
+  approve-release:
+    machine: true
+    steps:
+      - checkout
+      - run: python .circleci/approve-release.py
 
-  grakn-release-build:
+  release-cleanup:
+    machine: true
+    steps:
+      - checkout
+      - run: git push --delete origin trigger-ci-release
+
+  publish-github-draft:
     machine: true
     working_directory: ~/grakn
     steps:
       - checkout
       - bazel_install
-      - run: echo 'export GRAKN_VERSION=$(echo ${CIRCLE_TAG} | cut -c 2-)' >> $BASH_ENV # TODO: remove 'v' from tag name and use it as Grakn Version to be released
-      - run: mvn --batch-mode install -T 2.5C -DskipTests=true
-      - run: mkdir ~/grakn/artifacts
-      - run: mv grakn-dist/target/grakn-core-${GRAKN_VERSION}.zip ~/grakn/artifacts
-      - persist_to_workspace: # TODO: share Grakn with other jobs by putting it in the workspace
-          root: ~/grakn
-          paths:
-            - ./artifacts
-            - ./workbase
+      - bazel_add_rbe_credential
+      - run: bazel run //:deploy-github-zip -- ${GRABL_CREDENTIAL}
+
+  docs-auto-merge-to-master:
+    machine: true
+    steps:
+      - checkout
+      - run: |
+          mkdir docs
+          cd docs
+          python ../.circleci/docs-auto-merge-to-master.py
+
+  deploy-docs:
+    machine: true
+    steps:
+      - checkout
+      - run: |
+          mkdir docs
+          cd docs
+          python ../.circleci/deploy-docs.py
 
 workflows:
   ci-pull-request:
     jobs:
       - build:
           filters:
             branches:
-              ignore: master
+              ignore: /master|trigger-ci-release/
       - client-java:
           filters:
             branches:
-              ignore: master
+              ignore: /master|trigger-ci-release/
       - common:
           filters:
             branches:
-              ignore: master
+              ignore: /master|trigger-ci-release/
       - console:
           filters:
             branches:
-              ignore: master
+              ignore: /master|trigger-ci-release/
       - graql:
           filters:
             branches:
-              ignore: master
+              ignore: /master|trigger-ci-release/
       - server:
           filters:
             branches:
-              ignore: master
+              ignore: /master|trigger-ci-release/
       - test-integration:
           filters:
             branches:
-              ignore: master
+              ignore: /master|trigger-ci-release/
       - test-integration-reasoner:
           filters:
             branches:
-              ignore: master
+              ignore: /master|trigger-ci-release/
       - test-integration-analytics:
           filters:
             branches:
-              ignore: master
+              ignore: /master|trigger-ci-release/
```

Comment:
Check if possible to use references to trigger all tests without the need of repeating every command:
```
ci-pull-request:
    jobs:
      - reference_here
          filters:
      	      branches:
                 ignore: /master|trigger-ci-release/
```

Reply:
Fixed.

Using reference wasn't possible because the workflow in `ci-pull-request` and `ci-master-branch` isn't exactly the same - they're mostly the same but needs to ignore different branches in the filter.

The solution is to just have one workflow `ci` which is executed on `master` and pull requests where the approve-release and sync docs jobs are only executed in `master`